x-pack/plugins/security_solution/common/api/entity_analytics/entity_store/entities/common.schema.yaml

```diff
@@ -9,11 +9,11 @@ components:
     UserEntity:
       type: object
       required:
-        - "@timestamp"
+        - '@timestamp'
```

Comment:
just curious, did eslint rules change for this?

Reply:
I don't belive that `eslint` rules run on `yaml` files

Reply:
I have a formatter enabled 😅 
```
    "[yaml]": {
        "editor.defaultFormatter": "redhat.vscode-yaml"
    },
```

Reply:
I undid the `"` changes.